portfolio/src/main/webapp/comments.js

```diff
@@ -93,7 +96,22 @@ function createCommentElement(text) {
   return commentElement;
 }
 
-/** Posts a comment to the server */
+/** 
+ * Create the container for the timestamp of individual posts
+ * @param {int} postTime in milliseconds
+ * @return {HTMLElement} the html element containing the @param postTime
+ */
+function createPostTimeElement(postTime) {
+  let postTimeElement = document.createElement('small');
+  let postTimeString = '\n' + new Date(postTime).toLocaleString();
```

Comment:
https://stackoverflow.com/questions/13274279/authentication-filter-and-servlet-for-login/13276996#13276996